### PR TITLE
fix(archive): Headline and container values for images in grid view

### PR DESCRIPTION
**Pull request by sjunaid**
SD-1454

### CODE REVIEW

**Comment by eozan** [comment]
:+1:

File: client/app/scripts/superdesk/filters.js [hunk 1]
```diff
@@ -77,6 +77,11 @@ define([
                 }
             };
         })
+        .filter('truncateString', function() {
+            return function(inputString, limit, postfix) {
+                return _.trunc(inputString, {'length': limit, 'omission': postfix});
```
Comment (amagdas) [inline client/app/scripts/superdesk/filters.js]: We can supply a default value for 'omission' so we don't have to supply a value for it each time...

**Comment by amagdas** [comment]
:+1:

**Comment by mugurrus** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**